pkg/api/login_oauth.go

```diff
@@ -232,6 +274,13 @@ func syncUser(ctx *models.ReqContext, token *oauth2.Token, userInfo *social.Basi
 		}
 	}
 
+	return extUser
+}
+
+// syncUser syncs a Grafana user profile with the corresponding OAuth profile.
+func syncUser(ctx *models.ReqContext, extUser *models.ExternalUserInfo,
+	connect social.SocialConnector) (*models.User, error) {
```

Comment:
Preference: When you need multiple lines for function declarations and calls in Go, it's quite common to put each variable on it's own line.

```suggestion
func syncUser(
	ctx *models.ReqContext,
	extUser *models.ExternalUserInfo,
	connect social.SocialConnector,
) (*models.User, error) {
```